backend/app/services/share/share_webhook.py

```
@@ -0,0 +1,284 @@
+            start_time=now.isoformat(),
+            end_time=now.isoformat(),
+            description=description,
+            status=status,
+            detail_url=action_url,
+        )
+
+        # Send notification synchronously
+        return webhook_notification_service.send_notification_sync(notification)
+
+    except Exception as e:
+        logger.exception(f"Error sending share review notification: {e}")
```

Ruff (0.14.14)

[warning] Redundant exception object included in `logging.exception` call

(TRY401)